default.nix

```diff
@@ -96,5 +96,45 @@ rec {
     inherit plutus marlowe-playground plutus-playground;
   };
 
-  deployment-shell = pkgs.callPackage ./deployment/shell.nix { };
+  # This builds a vscode devcontainer that can be used with the plutus-starter project (or probably the plutus project itself).
```

Comment:
Could we perhaps move this whole block out of `default.nix` ? Either combining it with nix/devcontainer.nix or have a default.nix and a build-image.nix or whatever..

Reply:
Hmm, sure. It doesn't have too many dependencies now, it won't be too bad.

Reply:
Done.